imports/plugins/core/orders/server/no-meteor/queries/ordersByAccountId.js

```diff
@@ -0,0 +1,70 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+
+/**
+ * @name ordersByAccountId
+ * @method
+ * @memberof Order/NoMeteorQueries
+ * @summary Query the Orders collection for orders made by the provided accountId and (optionally) shopIds
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} params - request parameters
+ * @param {String} params.accountId - Account ID to search orders for
+ * @param {String} params.orderStatus - Workflow status to limit search results
+ * @param {String} params.shopIds - Shop IDs for the shops that owns the orders
+ * @return {Promise<Object>|undefined} - An Array of Order documents, if found
+ */
+export default async function ordersByAccountId(context, { accountId, orderStatus, shopIds } = {}) {
+  const { accountId: contextAccountId, collections, shopsUserHasPermissionFor, userHasPermission } = context;
+  const { Orders } = collections;
+
+  if (!accountId) {
+    throw new ReactionError("invalid-param", "You must provide accountId arguments");
+  }
+
+  let query = {
+    accountId
+  };
+
+  // Limit which orders are returned: `all`, `canceled`, `completed`, or open`
+  if (orderStatus !== "all" || null || undefined) {
```

Comment:
JS logic doesn't work this way. You would need `orderStatus !== "all" || orderStatus === null || orderStatus === undefined`, but I don't think you actually need the outer `if` at all. You can remove it and do `if/else if` or a `switch` on `orderStatus`.